scripts/badger/bridge_out_arbitrum_fees/bridge_out_erc20.py

```diff
@@ -0,0 +1,59 @@
+from rich.console import Console
+from brownie import interface, accounts, Contract
+from helpers.addresses import registry
+
+CONSOLE = Console()
+
+# tokens to bridge out
+tokens_out = ["CRV", "USDT", "SUSHI", "WBTC"]
```

Comment:
isnt there also a lot of $weth being withdrawn from the lp positions?

Reply:
and some $swapr?

Reply:
weth will stay in Arb for gas ops, i think we can send to techops and unwrap there

will add some method to swap `SWPR` for another asset

Reply:
thinking that for ops cost the value wd in `weth` is going to be quite a lot considering the tx fee in Arb

perhaps worth sending some to techops and rest to mainnet?

Reply:
yea techops makes sense for gas